### ISSUE TITLE
Comment is parsed as string after comments parsing is introduced

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Wire#515]
Triggered by following snippet:

```
  routingRules: {
    rule1: {
      pattern: '@Symbol =~ "[A-L].*"', # M intentionally missing
      policy: ROUND_ROBIN,
      routes: [ "INT1" ]
    }
}
```

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.23ea21](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.23ea21), [BOM-2.23ea75](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea75)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.23ea23](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.23ea23), [BOM-2.23ea80](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea80)
